pages/api/auth/[...nextauth].js

```diff
@@ -0,0 +1,12 @@
+import NextAuth from "next-auth/next";
+import GithubProvider from "next-auth/providers/github";
+
+export const authOptions = {
+  providers: [
+    GithubProvider({
+      clientId: process.env.GITHUB_ID,
```

Comment:
export environment variables from `./config/serverConfig.js` so it can be shared throughout the repo

Reply:
done